### PR TITLE
fix(swap): limit price on the confirmation screen

### PR DESCRIPTION
**Pull request by nistadev**
**Description**
While we had selected "Limit order" and we try to edit the price, if we press the delete button or any number on keyboard, several zeros are added/removed and the value does not update the token quantities. This PR fixes it.

**Screenshots**
image
image

### CODE REVIEW

**Review by Nebyt — DISMISSED** [review]

File: packages/yoroi-extension/app/components/swap/SwapPriceInput.js [hunk 1]
```diff
@@ -17,31 +17,39 @@ import {
...
 
-  if (swapStore.limitOrderDisplayValue === '' && marketPrice != null) {
-    runInAction(() => {
-      swapStore.setLimitOrderDisplayValue(formattedPrice);
-    });
-  }
-  const displayValue = isMarketOrder ? formattedPrice : swapStore.limitOrderDisplayValue;
+  // if (limitPrice.displayValue === '' && marketPrice != null) {
+  //   onChangeLimitPrice(formattedPrice);
+  // }
```
Comment (zuzunker) [inline packages/yoroi-extension/app/components/swap/SwapPriceInput.js]: @nistadev , this can be removed?

Comment (nistadev) [reply packages/yoroi-extension/app/components/swap/SwapPriceInput.js]: Yes! Theoretically 🤔

Comment (zuzunker) [inline packages/yoroi-extension/app/components/swap/SwapPriceInput.js]:
```suggestion
```

**Review by zuzunker — DISMISSED** [review]
/check

**Review by zuzunker — APPROVED** [review]

**Pull request merged**